src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/System.ServiceProcess.ServiceController.Tests.csproj

```diff
@@ -30,6 +30,7 @@
       <Project>{ceb0775c-4273-4ac4-b50e-4492718051ae}</Project>
       <Name>System.ServiceProcess.ServiceController.TestNativeService</Name>
       <SetConfiguration>Configuration=$(ConfigurationGroup)</SetConfiguration>
+      <SetPlatform>Platform=x64</SetPlatform>
```

Comment:
32-bit Windows is in our support matrix.  I don't know if that's just "execute", or "build and execute", but there is an x86 story.